sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/UninstallResourceCleaner.java

```diff
@@ -0,0 +1,48 @@
+package com.mesosphere.sdk.offer;
```

Comment:
Good idea. OK if I move both of these over as part of the separate wrap-up work? I hate to waste a green CI.